src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 logging.level.root= WARN
-logging.level.pl.put.poznan.transformer= DEBUG
+logging.level.pl.put.poznan.madness= DEBUG
+server.error.include-message=always
```

Comment:
To włącza `message` w przypadku wystąpienia `Exception` podczas requestu. Nie ma tam stack trace, więc wg mnie może zostać